### PR TITLE
added dataset mt

### PR DESCRIPTION
**Pull request by seankross**
Is this all I need to do to add a new datatset? I then tried to use `mt` to create a scatter plot with mpg on the X axis and hp on the Y axis and it only showed one point in the graph. Also the show table block is not wide enough to show the whole table.

### CODE REVIEW

File: src/components/blockly/fields/BuiltinDataSetField.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import postLoadCSV from "../dsl/workers/csv.proxy"
 
 const builtins = {
     dummy: withPrefix("/datasets/dummy.csv"),
+    mt: withPrefix("/datasets/mt.csv"),
```
Comment (pelikhan) [inline src/components/blockly/fields/BuiltinDataSetField.tsx:8]: probably want to use a descriptive name for the data set

**Comment by pelikhan** [comment]
If you look at the javascript console output, it will surface parsing errors on the CSV.

**Comment by pelikhan** [comment]
image

**Review by pelikhan — APPROVED** [review]

**Comment by pelikhan** [comment]
Also watch out that merge branches are automatically deleted... always start a new branch

File: src/components/blockly/fields/BuiltinDataSetField.tsx (hunk 1 above)
Comment (seankross) [reply src/components/blockly/fields/BuiltinDataSetField.tsx:8]: I agree, this was just a test to see if I can add something small.

**Comment by pelikhan** [comment]
Looks good ready to merge.

**Pull request merged**